src/main/java/Storage.java

```diff
@@ -0,0 +1,67 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileOutputStream;
+
+/**
+ * Utility class to help write to and modify .txt files
+ */
+public class Storage {
+
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Appends task description to a .txt file
+     *
+     * @param task
+     * @throws IOException
+     */
+    public void writeToFile(Task task) throws IOException {
+        FileWriter fw = new FileWriter(this.filePath, true);
+        if (task instanceof Todo) {
+            fw.write(((Todo) task).getIdentifier() + " | " + (task.getDone() ? 1 : 0) + " | " + task.getDescription() + "\n");
+        } else if (task instanceof Deadline) {
+            fw.write(((Deadline) task).getIdentifier() + " | " + (task.getDone() ? 1 : 0) + " | " + task.getDescription() + "| " + ((Deadline) task).getDate() + "\n");
+        } else {
+            fw.write(((Event) task).getIdentifier() + " | " + (task.getDone() ? 1 : 0) + " | " + task.getDescription() + "| " + ((Event) task).getDate() + "\n");
```

Comment:
Perhaps another method can be created to contain ```" | " + (task.getDone() ? 1 : 0) + " | " + task.getDescription() + "| "``` to make the code more readable?